save-agent/src/nativeMain/kotlin/org/cqfn/save/agent/SaveAgent.kt

```diff
@@ -140,6 +141,15 @@ class SaveAgent(private val config: AgentConfiguration,
         body = executionLogs
     }
 
+    /**
+     * Save agent version
+     */
+    private suspend fun sendAgentVersion() = httpClient.post<HttpResponse> {
```

Comment:
if we are keeping it as a separate query, please change it to something like `saveAdditionalData`, because we may add other info in there